pkg/client/helpers_test.go

```diff
@@ -0,0 +1,20 @@
+package client_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	william "github.com/topfreegames/Will.IAM/pkg/client"
+)
+
+func TestGenerateNil(t *testing.T) {
```

Comment:
Not sure we need to call these helper functions as `Test..` as they are not tests themselves. I suggest renaming them to `func BuildIlPermission` and `func BuildNilInfo()`. WDYT?